service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1632,6 +1661,29 @@ components:
             id:
               $ref: "#/components/schemas/Literal"
 
+    CriteriaCountQuery:
+      type: object
+      description: Count output entity instances for a search criteria
+      properties:
+        criteriaGroupSection:
+          $ref: "#/components/schemas/CriteriaGroupSection"
```

Comment:
If you're not going to take a single criteria you should take a list of sections so that an entire temporary cohort can be queried.

Reply:
Not against a list. Was going by existing api that takes sectionId. 
How about section, group and single criteria all in the body- similar to existing one that takes sectionid, or groupId

Reply:
The existing API takes single ids because it's calculating the count for just a part of the existing cohort. There's no reason for this API to take a partial cohort, since the caller is choosing what to include anyways, so it should either take a list of sections to allow a full cohort for flexibility or just a single criteria to do what we need right now. Sections include groups which include criteria so there's no need for them to be separate parameters.

Reply:
sgtm. If adding list of sections others are not needed

Reply:
added